gapis/replay/manager.go

```diff
@@ -153,13 +162,32 @@ func (m *manager) createScheduler(ctx context.Context, device bind.Device) {
 	m.mutex.Lock()
 	defer m.mutex.Unlock()
 	m.schedulers[deviceID] = scheduler.New(ctx, deviceID, m.batch)
+	ch := make(chan executeArgs, 16)
```

Comment:
This channel should be size zero. We want batching to still happen. If there is a replay being executed on the device, and we've already finished generating the next one, it is not advantageous to generate a 3rd one. Setting the size to zero, will still allow the second payload to be generated, but it will then block and wait for the first replay to finish before returning control to the generator, thus 3rd and subsequent requests can still be batched.